<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pre</class>
 <widget class="QDialog" name="Pre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>521</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:grey;
}</string>
      </property>
      <property name="text">
       <string>患者姓名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_patientname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:grey;
}</string>
      </property>
      <property name="text">
       <string>症状</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_symp"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:grey;
}</string>
      </property>
      <property name="text">
       <string>医生姓名</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_docname"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:grey;
}</string>
      </property>
      <property name="text">
       <string>医嘱</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:grey;
}</string>
      </property>
      <property name="text">
       <string>处方</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_prescription"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_advice"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_connect">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color:white;
	background-color:#22BD9E;
	qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string>病历填写</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>110</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color:white;
}</string>
   </property>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	background-color:#22BD9E;
	qproperty-alignment: 'AlignCenter';
	border: none;             /* 移除边框 */
    /*background: none;         /* 去掉背景色 */
	border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>提交病历</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_connect</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
